src/seedu/addressbook/data/person/Person.java

```diff
@@ -25,6 +27,13 @@ public Person(Name name, Phone phone, Email email, Address address, UniqueTagLis
         this.email = email;
         this.address = address;
         this.tags = new UniqueTagList(tags); // protect internal tags from changes in the arg list
+        if (this.sequenceNumber == 0) {
+        	this.sequenceNumber = 1;
+        	nextSequenceNumber = this.sequenceNumber + 1;
+        } else {
+        	this.sequenceNumber = nextSequenceNumber;
+        	nextSequenceNumber++;
+        }
     }
```

Comment:
Try to simply this to just `this.sequenceNumber = nextSequenceNumber++;` by making an appropriate initialization to the static variable.